packages/manager/apps/sign-up/src/form/form.controller.js

```diff
@@ -110,9 +111,12 @@ export default class SignUpFormAppCtrl {
     this.saveError = null;
     this.loading.init = true;
 
-    const { ovhSubsidiary } = this.$location.search();
-    if (ovhSubsidiary && ovhSubsidiary.match(/^[\w]{2}$/)) {
-      localStorage.setItem(OVH_SUBSIDIARY_ITEM_NAME, ovhSubsidiary);
+    this.$location.search();
+    this.isIndianSubsidiary = this.user.ovhSubsidiary;
```

Comment:
This is still not what is need, you need to check if ovhSubsidiary === 'IN' (of course do not use 'IN' but rather a constant)

Reply:
updated to search for a constant